portal-web/test/functional/com/liferay/portalweb/macros/frontendinfrastructure/DataSetAdmin.macro

```diff
@@ -650,6 +686,24 @@ definition {
 			value1 = ${key_type});
 	}
 
+	macro selectViewCreated {
```

Comment:
I would change the name of this macro, because "created" does not give us important info. You are trying to select a Data Set view in a Data Set fragment, so for me the name would be something like "selectDataSetFragmentView"

Reply:
Nice, done!!